public/images/azure.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 122.88 95.07" style="enable-background:new 0 0 122.88 95.07" xml:space="preserve"><style type="text/css">.st0{fill-rule:evenodd;clip-rule:evenodd;fill:#0089D6;}</style><g><path class="st0" d="M56.8,90.01l28.96-5.12l0.27-0.06L71.13,67.12c-4.94-5.83-10.17-11.99-14.79-17.83 c0-0.09,15.38-42.43,15.46-42.58C71.83,6.66,82.3,24.73,97.17,50.5l25.51,44.23l0.19,0.34l-47.33-0.01l-47.33-0.01l28.68-5.06 L56.8,90.01L56.8,90.01z M0,84.62c0-0.02,7.02-12.21,15.59-27.07l15.59-27.03l18.17-15.25C59.35,6.89,67.56,0.01,67.59,0 c-0.07,0.26-0.17,0.5-0.29,0.74L47.56,43.06L28.18,84.62l-14.09,0.02C6.34,84.66,0,84.64,0,84.62L0,84.62L0,84.62z"/></g></svg>
```

Comment:
this file should not be stored on the client but set via an absolute URL on from the server config

Reply:
I've removed the image, currently if an icon is provided it should be shown.